### DISCUSSION TITLE
Bugs ou Erros Pro - Versão: v2.4.2

### DISCUSSION THREAD

**Discussion by williamr8915** [cleitonme/izing.instalador#3] [Q&A]
Olá, utilizo a versão pro e estou com alguns problemas em dois itens especificos: 
Relátorios (não localizam nada e não exportam)
Flow com TypeBot - Não está funcionando

Não sou programador, entendo pouco mas o suficiente para realizar algumas alterações com direcionamento e a instalação, se alguem souber algo sobre estes dois pontos e puder auxiliar no ajuste, seria otimo.

**Comment by cleitonme** [comment]
Amigo essa versão não tem integração com typebot etc.. verifique com quem você comprou
